benchmarks/glyco/eval_yield.py

```
@@ -0,0 +1,55 @@
+pin, psms = sys.argv[1], sys.argv[2]
+
+def bare(s):
+    s = re.sub(r"\[[^\]]*\]$", "", s.strip())
+    if len(s) > 4 and s[1] == "." and s[-2] == ".": s = s[2:-2]
+    return re.sub(r"[^A-Za-z]", "", s).upper()
+
+hdr = None; rows = {}
+for ln in open(pin):
+    f = ln.rstrip("\n").split("\t")
+    if hdr is None:
+        hdr = f; iP = hdr.index("Peptide"); iL = hdr.index("Label")
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)